app/controllers/cyclists_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CyclistsController < ApplicationController
+  def new
+    @cyclist = Cyclist.new(User.new)
+  end
+
+  def show
+    @cyclist = Cyclist.new(User.find(params[:id]))
+  end
+
+  def create
+    @cyclist = Cyclist.new(User.new(cyclist_params))
+
+    if @cyclist.save
+      redirect_to cyclist_url(@cyclist)
```

Comment:
Since you're inheriting from some ActiveModel modules here, can you just redirect to the cyclist rather than use the URL helper?

If you can't simplify that, I try to use the `_path` rather than `_url` helpers when I'm not in a mailer to reduce friction if there's more than one app domain, etc.